src/_pytest/_py/error.py

```diff
@@ -90,15 +90,24 @@ def checked_call(
         except OSError as value:
             if not hasattr(value, "errno"):
                 raise
-            errno = value.errno
             if sys.platform == "win32":
                 try:
-                    cls = self._geterrnoclass(_winerrnomap[errno])
+                    cls = self._geterrnoclass(_winerrnomap[value.errno])
                 except KeyError:
                     raise value
             else:
                 # we are not on Windows, or we got a proper OSError
-                cls = self._geterrnoclass(errno)
+                # Not sure if we should do that:
+                # if errno is None:
```

Comment:
This seems reasonable to me if you want to do it.

Reply:
Hmm, I tricked myself by upgrading pre-commit in the last commit. So during rebase I was using an old version of mypy.